### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, lstat '/home/ivan/src/tp3-algo3/son'

### ISSUE THREAD

**Issue by FlyingPumba** [tualo/tualo-git-context#19]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.0.0
**System**: Ubuntu 14.04.2
**Thrown From**: [tualo-git-context](https://github.com/tualo/tualo-git-context) package, v0.6.10
### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat '/home/ivan/src/tp3-algo3/son'

```
At /usr/share/atom/resources/app.asar/src/pane.js:762

Error: ENOENT: no such file or directory, lstat '/home/ivan/src/tp3-algo3/son'
  at Error (native)
  at fs.lstatSync (fs.js:836:18)
  at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:170:16)
  at TualoGitContextView.module.exports.TualoGitContextView.gitStatus (/home/ivan/.atom/packages/tualo-git-context/lib/tualo-git-context-view.coffee:150:13)
  at /home/ivan/.atom/packages/tualo-git-context/lib/tualo-git-context-view.coffee:76:12
  at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextBuffer.module.exports.TextBuffer.saveAs (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:996:20)
  at TextBuffer.module.exports.TextBuffer.save (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:978:19)
  at TextEditor.module.exports.TextEditor.save (/usr/share/atom/resources/app.asar/src/text-editor.js:579:26)
  at Pane.module.exports.Pane.saveItem (/usr/share/atom/resources/app.asar/src/pane.js:523:18)
  at Pane.module.exports.Pane.saveActiveItem (/usr/share/atom/resources/app.asar/src/pane.js:506:19)
  at Workspace.module.exports.Workspace.saveActivePaneItem (/usr/share/atom/resources/app.asar/src/workspace.js:592:35)
  at atom-workspace.atom.commands.add.core:save (/usr/share/atom/resources/app.asar/src/workspace-element.js:311:30)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:180:20)

```
### Commands

```
     -1:01.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -0:57.6.0 editor:newline (atom-text-editor.editor.is-focused)
  5x -0:39.3.0 core:undo (atom-text-editor.editor.is-focused)
  2x -0:21.1.0 key-binding-resolver:toggle (atom-text-editor.editor.is-focused)
     -0:19.1.0 settings-view:open (atom-text-editor.editor.is-focused)
     -0:10.9.0 core:move-left (atom-text-editor.editor.is-focused)
     -0:09.6.0 core:delete (atom-text-editor.editor.is-focused)
     -0:09.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:07.9.0 core:save (atom-text-editor.editor.is-focused)
     -0:05.2.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:05.1.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:03.7.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:03.4.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:03.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:01.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:00.6.0 core:save (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "piatto-light"
    ],
    "disabledPackages": [
      "git-control",
      "git-plus",
      "git-tab-status",
      "language-latex",
      "language-x86",
      "merge-conflicts",
      "pdf-view",
      "tualo-git-context"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
git-tab-status, v1.9.2
language-latex, v0.6.1
language-x86, v0.1.0
latex, v0.24.2
latexer, v0.2.3
pdf-view, v0.22.0
piatto-light, v2.0.2
tualo-git-context, v0.6.10

# Dev
No dev packages
```

**Comment by thomashoffmann1979** [comment]
Hi, thank you for reporting. There was an unchecked variable, i have fixed it in version 0.6.11.

**Comment by FlyingPumba** [comment]
Glad to hear that !
